.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Java version**: x.y.z
+**Library version**: x.y.z
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**How to reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
this does not make sense for the library.

Reply:
Fixed